### ISSUE TITLE
[QUESTION] <Compatible with Puffish Skills?>

### ISSUE THREAD

**Issue by DivannKokos** [Darkhax-Minecraft/AttributeFix#52]
### Question

In config\attributefix.json I've noticed this strange range:
```
    "puffish_skills:player.ranged_damage": {
      "enabled": false,
      "min": {
        "default": 0,
        "value": 0
      },
      "max": {
        "default": 179769313486231570000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000,
        "value": 179769313486231570000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000
      }
    },
```
It is also this way with some other attributes of Puffish Skills
While the rest of other mods and vanilla more like this: (numbers vary but they are smaller than example above)
```
    "puffish_skills:player.stamina": {
      "enabled": false,
      "min": {
        "default": 0,
        "value": 0
      },
      "max": {
        "default": 1024,
        "value": 1024
      }
    },
```
So the question is, is the example above is okay, or something is broken, and shouldn't be like this.
TLDR: Is AttributeFix compatible with Puffish Skills?

**Comment by Darkhax** [comment]
Hello @DivannKokos the values shown in the config file are the values specified by the developer of Puffish Skills. In this case they chose that very long number which is the maximum possible value that can be used. This looks odd, but is not an issue for either mod.
